src/app/overview/overview.component.html

```diff
@@ -1,34 +1,117 @@
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Incremental analysis of Git repositories</h4><p class="">Incremental analysis of Git repositories</p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Fixpoint algorithms</h4><p class=""> Generic fixpoint algorithms with widening and narrowing.</p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Global invariant</h4><p class=""> Global invariant approach and side-effecting.</p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>HTML front-end</h4><p class=""> Visualisation of results.</p></div> -->
+
 </div>
+<br />
+<br />
+<br />
 <div class="row d-flex justify-content-center">
-    <h2>Overview of Goblint Features</h2><br>
+    <h2>Supported Properties</h2><br>
 </div>
 <div class="row d-flex justify-content-center">
-    <div class="col-md-3 col-sm-6 mt-5"><h4>Race detection</h4><p class=""> The main application of this framework is for race detection. 
-        Goblint checks  that all accesses to shared memory are protected by a common mutex.</p></div>
-    <div class="col-md-3 col-sm-6 mt-5"><h4>Path-sensitivity</h4><p class=""> Goblint is path-sensitive, so it deals with conditional locking schemes as well as possibly failing, 
-        interruptible and probing locking primitives. Mutexes may be handled both pathsensitively and symbolically.</p></div>
-    <div class="col-md-3 col-sm-6 mt-5"><h4>Thread-modularity</h4><p class=""> Goblint performs thread-modular analysis of C programs. 
-        It computes a sound upper approximation of all possible thread interleavings using global invariants for shared data.</p></div>
-    <!-- TODO: fill the feature descriptions 
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Race detection</h4><p class=""> Goblint started its life as a framework for race detection, and still excells here.
+        Goblint checks that all accesses to shared memory are protected by a common mutex, also taking into account which threads can execute in parallel, and which can't.</p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Reach safety</h4><p class=""> Goblint is capable of checking whether some assert is guaranteed to hold at a given program point,
+        or whether a certain program is unreachable. This can be used to encode almost any safety property of interest.
+    </p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Integer Overflows</h4><p class=""> Goblint can verify that programs are free of overflows of (signed) overflows of integers, which
+        are notorious causes of Undefined Behavior and bugs in real-world programs.
+    </p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>And many more...</h4><p class=""> Goblint comes with a wide variety of further analysis, and it is easy to extend for analysis engineers. It suffices
+        to define the domain and the right-hand sides and that's it! Here is a <a href="https://goblint.readthedocs.io/en/latest/developer-guide/firstanalysis/" target="_blank">tutorial</a> for extending Goblint.
```

Comment:
"Right-hand sides" lacks some background here, so maybe this instead?
```suggestion
        to define the domain and the transfer functions, and that's it! Here is a <a href="https://goblint.readthedocs.io/en/latest/developer-guide/firstanalysis/" target="_blank">tutorial</a> for extending Goblint.
```